### PR TITLE
Change PXF port number to 5888

### PR DESCRIPTION
**Pull request by lavjain**

### CODE REVIEW

File: gpAux/extensions/pxf/src/pxfutils.c [hunk 1]
```diff
@@ -17,15 +16,7 @@ normalize_key_name(const char *key)
...
-	StringInfoData formatter;
-
-	initStringInfo(&formatter);
-	char	   *upperCasedKey = str_toupper(pstrdup(key), strlen(key));
-
-	appendStringInfo(&formatter, "X-GP-%s", upperCasedKey);
-	pfree(upperCasedKey);
-
-	return formatter.data;
+	return psprintf("X-GP-%s", str_toupper(pstrdup(key), strlen(key)));
```
Comment (denalex) [inline gpAux/extensions/pxf/src/pxfutils.c]: do we need to remember the pointer to free it later ?

File: gpAux/extensions/pxf/src/pxfutils.c [hunk 2]
```diff
@@ -38,27 +29,17 @@ TypeOidGetTypename(Oid typid)
...
-
-	char	   *typname = NameStr(typform->typname);
-
-	StringInfoData tname;
-
-	initStringInfo(&tname);
-	appendStringInfo(&tname, "%s", typname);
-
+	Form_pg_type typform = (Form_pg_type) GETSTRUCT(typtup);
+	char	   *typname = psprintf("%s", NameStr(typform->typname));
```
Comment (denalex) [inline gpAux/extensions/pxf/src/pxfutils.c]: NameStr returns smith that needs to be freed ?

File: gpAux/extensions/pxf/src/pxfutils.c (hunk 1 above)
Comment (lavjain) [reply gpAux/extensions/pxf/src/pxfutils.c]: str_toupper() uses pnstrdup internally, which obviates the need to free it.

File: gpAux/extensions/pxf/src/pxfutils.c (hunk 2 above)
Comment (lavjain) [reply gpAux/extensions/pxf/src/pxfutils.c]: From src/include/c.h:
```
#define NameStr(name)	((name).data)
```

**Pull request merged**